### DISCUSSION TITLE
Can't use if I have chatgpt plus?

### DISCUSSION THREAD

**Discussion by TheFern2** [ztjhz/BetterChatGPT#624] [General]
I figured to ask this, so if I already have chatgpt plus, I would have to pay again for the api? Is there any sort of metrics if that's more expensive than the chatgpt monthly fee?

**Comment by saadeddinmd** [comment]
> I figured to ask this, so if I already have chatgpt plus, I would have to pay again for the api? Is there any sort of metrics if that's more expensive than the chatgpt monthly fee? 

API is different from chatgpt. These are completely two different things and what you pay for one cannot be used for the other.

API is like dealing with the manufacturer directly. You buy raw stuff and implement it the way you want. Chatgpt is a ready-to-use consumer product that openAI provide.

API is much cheaper usually, but eventually it depends on your usuage. For most cases, API is cheaper.

**Reply by TheFern2** [reply]
Thanks I figured that much was just hoping it would be different I guess I can stop paying for ChatGPT and try the api and see how it goes one thing is for sure I cannot stand the simple ChatGPT gui anymore.
